pydis_site/settings.py

```diff
@@ -230,7 +230,9 @@
     'DEFAULT_PERMISSION_CLASSES': (
         'rest_framework.permissions.DjangoModelPermissions',
     ),
-    'TEST_REQUEST_DEFAULT_FORMAT': 'json'
+    'TEST_REQUEST_DEFAULT_FORMAT': 'json',
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
```

Comment:
We don’t want to use pagination everywhere, only in the infraction route. Can you please revert this change? This also cause some tests to fail.

Reply:
Sure, I'll revert this one